packages/chrome/src/views/subnav/CollapsibleSubNavItem.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import classNames from 'classnames';
+import ChromeStyles from '@zendeskgarden/css-chrome';
+import { retrieveTheme } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.collapsable_sub_nav_item';
+const PANEL_COMPONENT_ID = 'chrome.collapsable_sub_nav_item_panel';
+
+import SubNavItem from './SubNavItem';
+
+/** Accepts all `<div>` props */
+const StyledSubNavItemHeader = styled(SubNavItem).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  'aria-expanded': props => props.expanded,
+  className: props =>
+    classNames(ChromeStyles['c-chrome__subnav__item--header'], {
+      [ChromeStyles['is-expanded']]: props.expanded
+    })
+})`
+  ${props => retrieveTheme(COMPONENT_ID, props)};
+`;
+
+StyledSubNavItemHeader.propTypes = {
+  expanded: PropTypes.bool
+};
+
+/** Accepts all `<div>` props */
+const StyledSubNavPanel = styled.div.attrs({
+  'data-garden-id': PANEL_COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  className: props =>
+    classNames(ChromeStyles['c-chrome__subnav__panel'], {
+      [ChromeStyles['is-hidden']]: props.hidden
```

Comment:
Does this need `aria-hidden`? What's your thinking on extracting some of this logic into an [Accordion](https://www.w3.org/TR/wai-aria-practices/#accordion) container?

Reply:
> Does this need aria-hidden

Since it has `display: none` it _shouldn't_ need it, but I've added it just in case 😄 

> What's your thinking on extracting some of this logic into an Accordion container?

Lets discuss some more offline. Normally an Accordion widget is in relation between several related panels & collapsable areas. Not really sure how this will all piece together in SubNav

Reply:
The WAI-ARIA pattern linked above applies to a single combination of an accordion `header` and its `panel` – exactly the pattern we have here (obviously, a Chrome subnav may have multiples of these). At a minimum, we should consider adding the expected aria `role` and `aria-controls` attributes.